entrypoint.sh

```diff
@@ -70,15 +88,15 @@ configure_kernel_module_locking() {
     popd
     sync
     umount "${mount_path}"
-    echo "Rebooting"
+    warn "Rebooting"
     echo b > /proc/sysrq-trigger
   fi
 }
 
 check_cached_version() {
-  echo "Checking cached version"
+  info "Checking cached version"
   if [[ ! -f "${CACHE_FILE}" ]]; then
-    echo "Cache file ${CACHE_FILE} not found."
+    warn "Cache file ${CACHE_FILE} not found."
```

Comment:
Should this be `warn`?

Reply:
On a second thought this could just be an `info` log.  Is that what you had in mind?

Reply:
Yes.

Reply:
Done